metric/response_size.go

```diff
@@ -0,0 +1,47 @@
+package metric
+
+import (
+	"fmt"
+	"net/http"
+
+	otelmetric "go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/semconv/v1.20.0/httpconv"
+)
+
+const (
+	metricNameResponseSizeBytes = "response_size_bytes"
+	metricUnitResponseSizeBytes = "bytes"
```

Comment:
Previously I use `By` for the unit as I follow metrics unit in [opentelemetry-go/semconv](https://github.com/open-telemetry/opentelemetry-go/blob/814a41388b9792d7389d9b631418bb28d3e0f2d3/semconv/v1.24.0/metric.go#L451). I know it weird they using `By` somehow, but lets us follow the convention as it also helps the exporter processing like [this](https://github.com/open-telemetry/opentelemetry-go/blob/814a41388b9792d7389d9b631418bb28d3e0f2d3/exporters/prometheus/exporter.go#L380). wdyt?